src/Nethermind/Nethermind.Blockchain/BlockTreeMethodOptions.cs

```diff
@@ -33,7 +33,7 @@ public enum BlockTreeLookupOptions
 }
 
 [Flags]
-public enum BlockTreeInsertOptions
+public enum BlockTreeInsertHeaderOptions
```

Comment:
All those options should be documented. It's getting extremely confusing what does what.

Reply:
100% agreed